recipes/xsel/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "xsel" %}
+{% set version = "1.2.1" %}
+{% set am_version = "1.15" %} # keep synchronized with build.sh
+
+package:
+  name: {{ name|lower }}

```

I prefer not to template the name; not like it could change anyway (also remove first line)
```suggestion
  name: xsel
```

---

Done.